src/styles/scss/templates/footer.scss

```diff
@@ -2,7 +2,6 @@
 
 .dc-c-footer {
   background-color: white;
-  padding-top: 100px;
```

Comment:
Moved this 100px to the bottom of main, as it was showing a weird line on sites that have a gray background instead of just white.